fn_useragentanalysis/fn_useragentanalysis/util/customize.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+
+"""Generate the Resilient customizations required for fn_useragentanalysis"""
+
+from __future__ import print_function
+from resilient_circuits.util import *
+
+def codegen_reload_data():
+    """Parameters to codegen used to generate the fn_useragentanalysis package"""
+    reload_params = {"package": u"fn_useragentanalysis",
+                    "incident_fields": [], 
+                    "action_fields": [], 
+                    "function_params": [u"user_agent_string"], 
+                    "datatables": [], 
+                    "message_destinations": [u"useragentanalysis_fn"], 
+                    "functions": [u"fn_useragentanalysis"], 
+                    "phases": [], 
+                    "automatic_tasks": [], 
+                    "scripts": [], 
+                    "workflows": [u"user_agent_workflow"], 
+                    "actions": [] 
```

Comment:
There should be an example run ("actions") as part of this distribution. You can add by doing:

resilient-circuits codegen --reload fn_useragentanalysis --rule "new rule"